moped-database/migrations/1650301717000_alter_project_list_view/up.sql

```diff
@@ -0,0 +1,89 @@
+drop view project_list_view;
+
+CREATE VIEW public.project_list_view AS
+with moped_team_members as (
+  select
+    mpp.project_id,
+    string_agg(DISTINCT concat(mu.first_name, ' ', mu.last_name, ':', mpr.project_role_name), ',') AS project_team_members
+   FROM 
+    public.moped_proj_personnel mpp
```

Comment:
The pattern of using very abbreviated table aliases is a common one in SQL constructions, so I may be an outlier here, but I've always found it easier to debug my statements when I return to them later if I alias long table names down to a real word, like:

`moped_projects` → `projects`
`moped_proj_personnel` → `personnel`
`moped_users` → `users`
`moped_project_roles` → `roles`

This is a personal style, so please don't adapt to it unless you find it useful in your own workflows!